en/getting_started/basic_flight_modes.md

```diff
@@ -2,185 +2,178 @@
 
 ![MC Position Mode](../../images/flight_modes/position_MC.png)
  
  
-## Autonomous Modes
+### Autonomous Modes
 
 Autonomous or Auto modes do not require any input from the pilot.
  
-**PAUSE: Auto Pause**
+#### Pause (Auto Pause)
 
 You can use Pause in case of emergency or during a mission in order to Hold the aircraft at its current position autonomously, while you get time to access the situation either emergency or mission related. When you activate Pause mode with a pre-programmed switch, the copter or multi-copter will fix itself to its current GPS position and altitude and hover there. During hold you are not able to control the craft with the RC. It is the first line of defense if you loose control over the aircraft and panic.
- 
-![GPS](../../images/flight_modes/GPS_s.png) **RETURN: Return to Launch (RTL) / Return to Home**
+
+#### Return Mode (Return to Launch/Home - RTL)
+
+![GPS](../../images/flight_modes/GPS_s.png)
 
 The copter or multicopter will return in a straight line at the current altitude (if higher than the home position + hovering altitude) or on the hovering altitude (if higher than the current altitude), then lands automatically.
- 
-![GPS](../../images/flight_modes/GPS_s.png) **MISSION**
 
-In Mission mode the aircraft will start the execution of a predefined Mission or Flight Plan. The Mission can be edited and uploaded to the aircraft with the software called Ground Control Station. The PX4 GCS is called QGroundControl and it can run on Windows, Linux, Android and iOS. You can download it from qgroundcontrol.com. The Mission upload to the aircraft can be done via USB connection on the ground or by wireless Data Link sometimes called Telemetry. During flight you can edit and change the current mission via this Data Link and monitor the current flight status and telemetry data live.
- 
-![GPS](../../images/flight_modes/GPS_s.png) **TAKE OFF**
+#### Mission Mode
 
-When **Take Off** is activated the multicopter automatically will give thrust to the motors such to take off then climb vertically to a preset take off altitude, stop the climb and hover at place witing for pilot's command input.
- 
-![GPS](../../images/flight_modes/GPS_s.png) **LAND**
+![GPS](../../images/flight_modes/GPS_s.png)
+
+In Mission mode the aircraft will start the execution of a predefined mission or flight plan. The mission can be edited and uploaded to the aircraft with a Ground Control Station (GCS).
+
+> **Tip** The PX4 GCS is called [QGroundControl](https://docs.qgroundcontrol.com/en/).
```

Comment:
@tubeme I took out some of your explanation here. It is too much detail - we need to have that detail in the "right" place - QGC docs or a QGC page in this doc set. TBD!

Reply:
Because I spent a lot of years in the Marketing field there is a golden rule regarding injecting information in peoples brain. It is called the Ladder principle, ie in order to have new information digested by a person, you have to make sure you relate this information to something that the user knows already. So sometimes my explanations might seem very detailed, but at the same time I write in this style for people that just start to fly and have no knowledge about anything related to PX4, flying or drones. So read the doc with this perspective. Of course for the advanced user this might seem little too detailed.

Reply:
To some extent I am a beginner, so I am always coming to this as "not an advanced user" :-) Ladder principle definitely valid in technical writing, but not so useful when targeting an audience that doesn't have much background to build upon.

In this case the user needs to know that mission mode executes autonomous missions. They also need to know that missions are created in a GCS and uploaded. 
They don't need to know that the GCS can run on any particular hardware or how QGC is connected to the vehicle. At this point that information is irrelevant to understanding Flight modes and therefore distracting.

Overall I think that this is a really good introduction - addressing the audience really well. In particular the icons and images are awesome. What you're mostly likely to see from me is simplification of English. If I do remove information you think is relevant then very happy to discuss. I am a beginner with these modes!